index.html

```diff
@@ -2,80 +2,1625 @@
 <html lang="en">
 
 <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
-        integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
-    <link rel="stylesheet" type="text/css" href="style.css">
-    <link href="https://fonts.googleapis.com/css?family=Archivo+Narrow:500|Noto+Sans+KR:900&display=swap"
-        rel="stylesheet">
-    <script src="https://kit.fontawesome.com/cbc2432238.js"></script>
-    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css"
-        integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+    integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@700;800&display=swap" rel="stylesheet">
+  <link rel="stylesheet" type="text/css" href="style.css">
+  <script src="https://kit.fontawesome.com/cbc2432238.js"></script>
+  
+  <link rel="icon" type="image/png"
+    href="https://s.newsweek.com/sites/www.newsweek.com/themes/newsweek/favicons/favicon-32x32.png" sizes="32x32">
+
+  <title>Newsweek - News, Analysis, Politics, Business, Technology</title>
+</head>
 
 <body>
-    <div class="fixed-top" id="stickynav">
-        <div class="d-flex justify-content-between" id="navscroll">
-            <div class="p-2">
-                <date id="datenav">Tue, Aug 6, 2019</date>
+  
+  <header>
+<div class="row navbar sticky-top navbar-expand-sm header-bg navbar-dark bg-red d-flex justify-content-between">
+      <p class ="header-date text-white text-decoration-none text-capitalize">Tue,  Aug 6,  2019</p>
+    <h1 class="newsweeklogo">Newsweek</h1>
+    <div class="order-on-small-screen">
+     <a href="#" class="nw-btn text-white p-2 text-uppercase text-decoration-none font-weight-bold">
+                Sign in</a>
+            <a href="#" class="nw-btn-white sub p-2 text-danger text-uppercase text-decoration-none bg-white font-weight-bold">
+                Subscribe
+                <i class="fas fa-chevron-right text-danger" id="shevron"></i>
```

Comment:
- [ ] I can 't see the red here. Please fix that as well.

![image](https://user-images.githubusercontent.com/61479648/88110610-de100400-cb82-11ea-8900-698739c9ab2f.png)